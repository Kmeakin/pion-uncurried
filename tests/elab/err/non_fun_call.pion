let f = false;

let x = f(false);
