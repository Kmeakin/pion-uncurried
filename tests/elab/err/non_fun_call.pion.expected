let f: Bool = false;

let x: #error = #error;

====DIAGNOSTICS====
Error: Unable to infer type of definition
   ╭─[tests/elab/err/non_fun_call.pion:3:5]
   │
 3 │ let x = f(false);
   ·     │ 
   ·     ╰─ here
───╯
Error: Called non-function expression
   ╭─[tests/elab/err/non_fun_call.pion:3:9]
   │
 3 │ let x = f(false);
   ·         │ 
   ·         ╰─ Help: type of this expression is `Bool`
───╯
